ci.properties

```diff
@@ -4,8 +4,8 @@ FAB_BASE_VERSION=2.0.0
 FAB_BASEIMAGE_VERSION=0.4.15
 # Pull below list of images from Hyperledger Docker Hub
 FAB_THIRDPARTY_IMAGES_LIST=kafka zookeeper couchdb
-# Set compaitable go version
-GO_VER=1.12.5
+# Set compatible go version
+GO_VER=1.12.9
```

Comment:
This doesn't hold up the PR but, is this still needed at all?

Reply:
I'm not sure, I'm going to look at the daily and release jobs and see where we might use it there. I'm working to see where these vars might be referenced in existing CI